README.md

```diff
@@ -203,7 +203,6 @@ package:
 * TAP
 * JUnit
 * nagios - Nagios/Sensu compatible output /w exit code 2 for failures.
-* nagios_verbose - nagios output with verbose failure output.
```

Comment:
Nah, just felt the readme should be at a higher level. It's still in the manual.